foundation_cms/templates/patterns/pages/nothing_personal/nothing_personal_base.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
```

Comment:
I've created this new file following the updated structure from the model renaming work in PR #14497